src/test/rpc/JSONRPC_test.cpp

```diff
@@ -2060,6 +2060,54 @@ class JSONRPC_test : public beast::unit_test::suite
             result[jss::result][jss::request][jss::command] == "bad_command");
     }
 
+    void
+    testAutoFillFailsLocalCheck()
```

Comment:
This test name feels incorrect? This autofill is successful, it looks like

Reply:
The fee should be much higher than base. It "fails" here: https://github.com/XRPLF/rippled/blob/91a54023b0a7a133f8d9ec2950c7dd386f4ee2ba/src/xrpld/rpc/detail/TransactionSign.cpp#L780

I can throw, but it will still end up with the reference_fee unless you want me to change all of that functionality.

I can also change the name of the test if you prefer?

Reply:
Yeah, the name is a bit confusing (+ a comment basically saying this would be useful).

Maybe `CheckFeeFailsLocalChecks`, since it's not technically called "autofill" in rippled?